shared/js/actions.js

```diff
@@ -131,6 +131,27 @@ loop.shared.actions = (function() {
     }),
 
     /**
+     * Used to send a message to the other peer.
+     */
+    SendCursorPosition: Action.define("sendCursorPosition", {
+      type: String,
```

Comment:
If we're including the `type` in the object, the name action should probably be more of `SendCursorData`